fineract-provider/src/main/java/org/apache/fineract/portfolio/loanaccount/rescheduleloan/service/LoanRescheduleRequestWritePlatformServiceImpl.java

```diff
@@ -459,6 +461,7 @@ public CommandProcessingResult approve(JsonCommand jsonCommand) {
             // update the loan object
             loan = saveAndFlushLoanWithDataIntegrityViolationChecks(loan);
             postJournalEntries(loan, existingTransactionIds, existingReversedTransactionIds);
+            loanAccrualTransactionBusinessEventService.raiseBusinessEventForAccrualTransactions(loan, existingTransactionIds);
```

Comment:
Please raise the event after the "recalculateAccruals" method which might created new Accrual transaction as well.